clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -48,7 +48,16 @@ object UncommittedGarbageCollector {
     val secretKey = hc.get(LAKEFS_CONF_API_SECRET_KEY_KEY)
     val connectionTimeout = hc.get(LAKEFS_CONF_API_CONNECTION_TIMEOUT_SEC_KEY)
     val readTimeout = hc.get(LAKEFS_CONF_API_READ_TIMEOUT_SEC_KEY)
+    val skipSecondsStr = hc.get(LAKEFS_CONF_DEBUG_GC_UNCOMMITTED_SKIP_SECONDS_KEY)
```

Comment:
I think we should have a minimal duration requirement - setting it too low may result in a serious bug.
It should be aligned somehow with the link physical address expiry time

Reply:
The default value should be aligned, I agree. Do you see any value (in terms of "safety") in a minimum value that is bigger than a minute? We need to add system-tests and waiting more than that doesn't sound reasonable.

Reply:
I don't think it's wise to let the users define the expiry for the tokens for non-testing purposes. For example, setting it to a low number in the server may lead to blocking all writes due to invalidity of all tokens.

Also, if the user is messing with arguments that are labeled as `debug` in a task that is deleting data, he can expect undefined behaviour from it.

Reply:
I didn't mean the users will be able to modify it in the server. My idea was to provide this parameter from the server. This way we'll be able to control it via the tests and validate it against the expiry token

Reply:
How would you change it from the server in a way that you can and the users can't?

Reply:
In unit tests that should not be an issue. For system tests we could add an undocumented configuration parameter, but once it's in the server, we can always validate it against the token timeout and handle conflicting values. Leaving it in the client side it means we will have no way to identify this problem

Reply:
Opened #4695 to tackle this.